### PR TITLE
Portfolio Project 1: Setup first 2 secctions (Mobile Skeleton)

### PR DESCRIPTION
**Pull request by Leon-Espitia**
In this Milestone I:

- Replicate Figma portfolio Template 1.
- Added HTML and CSS files.
- Run HTML&CSS linters.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,31 @@
 # Portfolio-project-set-up-and-mobile-version-skeleton
 Project: To parse 2 sections from a Figma portfolio design.
+
+# Project Name
+
+- Setup and mobile version skeleton
```
Comment (omar-muhamad) [inline README.md]: - [x] Please remove that because you already have the project name and description above.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,31 @@
...
+
+
+🤝 Contributing
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the issues page.
+
+## Show your support
+Give a ⭐️ if you like this project!
+
```
Comment (omar-muhamad) [inline README.md:26]: - [ ] Please add a `license` section here at the end of your Readme file with the `MIT` open software license like the Microverse Readme template.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Figma Template 1</title>
+    <link rel="stylesheet"
+          href="https://fonts.googleapis.com/css2?family=Crimson+Pro">
+    <link rel="stylesheet" href="style.css">   
+</head>
+<body>
+    <header>
+        <div class="logo"><a>Leon Espitia</a></div>
```
Comment (omar-muhamad) [inline index.html]: - [x] Please remove the Wrapping `<div>` tag and put your logo link `<a>Leon Espitia</a>` directly on your header for semantic reasons.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+              <a href="" target="_blank">
+                <img src="./assets/icons/M.svg" alt=""/>
+              </a>
+              <a href="" target="_blank">
+                <img src="./assets/icons/Github.svg" alt=""/>
+              </a>
+              <a href="" target="_blank">
+                <img src="./assets/icons/Fin.svg" alt=""/>
+              </a>
+          </div>
```
Comment (omar-muhamad) [inline index.html]: - [ ] Please make sure that The list of icons uses a `<ul>` tag to group them, and `<li>` per icon; each icon has `<a>` link tag. for example:
```
<ul>
    <li>
        <a href="" target="_blank">
             <img src="./assets/icons/Twiter.svg" alt=""/>
        </a>
    </li>
    .................
    .................
</ul>
```

File: style.css [hunk 5]
```diff
@@ -0,0 +1,64 @@
+body {
+  font-family: Arial, Helvetica, sans-serif;
+  background: url(./assets/backgrounds/header-background-mobile.png);
+  background-color: #e5e5e5;
+  flex-direction: column;
+  position: absolute;
+  width: 375px;
+  height: 718px;
+  margin: 24, 206, 24, 362;
+}
```
Comment (omar-muhamad) [inline style.css]: - [x] Please add more left and right padding to all page elements to match the given design.

| Figma Design  | Your Design |
| ------------- | ------------- |
| image | image|

**Review by omar-muhamad — CHANGES_REQUESTED** [review]
# Changes Required ♻️
### Hi @Leon-Espitia,

Nice job, you are almost there.



## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- Github flow is used correctly. ✔
- All linters checks are ok. ✔

## (Changes Required) Aspects to improve: ♻️

- [x] Please check the inline comments below.

## [Optional] suggestions:

- Nothing to mention.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html (hunk 4 above)
Comment (Leon-Espitia) [reply index.html]: Hi! Omar-Muhamad
I really appreciate all your comments and suggestions.
Thank you.

**Review by ShahierNashaat — APPROVED** [review]
Hi @Leon-Espitia , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Approved :ok_hand: :ok_hand: :ok_hand:

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**